src/instructlab/utils.py

```diff
@@ -260,7 +268,6 @@ def _load_schema(path: "importlib.resources.abc.Traversable") -> "referencing.Re
     """
     # pylint: disable=import-outside-toplevel
     # Third Party
-    from referencing import Resource
     from referencing.exceptions import NoSuchResource
     from referencing.jsonschema import DRAFT202012
```

Comment:
I am happy to do it. But this would have nothing to do with mypy enablement. I will make a note to follow up with cleaning up these pylint exceptions separately.